sanic/app.py

```diff
@@ -28,15 +28,17 @@ class Sanic:
 
     def __init__(self, name=None, router=None, error_handler=None,
                  load_env=True, request_class=None,
-                 strict_slashes=False, log_config=None):
+                 strict_slashes=False, log_config=None, 
```

Comment:
remove the extra space here: `strict_slashes=False, log_config=None, ` 

travis message: `sanic/app.py:31:56: W291 trailing whitespace`